eclipse-solargraph-plugin/src/main/java/io/github/pyvesb/eclipse_solargraph/utils/GemHelper.java

```diff
@@ -26,11 +26,17 @@
 
 public class GemHelper {
 
+	private static String buildPlatformCmd(String cmd, String gem) {
```

Comment:
Would naming this method as `buildGemCmd` be more appropriate? There is nothing platform-specific about it.

Reply:
If you prefer so... Just named it the way it is because `getPlatformCommand` was around already and `buildCmd` read a bit too short and unspecific.